.github/pull_request_template.md

```diff
@@ -5,31 +5,22 @@
 
 ### Request Reviewer
 > You can add desired reviewers here with an @mention.
-> For a full list of reviewers check out the *Committers* slot on [the project site](https://projects.eclipse.org/projects/technology.sw360.antenna/who).
-> A list of their GitHub names can be found in the [Eclipse Antenna Wiki](https://github.com/eclipse/antenna/wiki/Current-Committers).
 
 ### Type of Change
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Improvements
-- [ ] Documentation update
-- [ ] CI
-- [ ] Other
+> Mention one of the following:   
+> bug fix | new feature | improvements | documentation update | CI | Other
+
+*Type of change*:  
 
 ### How Has This Been Tested?
-- [ ] Specific Unit Test
-- [ ] Executing the Example Projects
-- [ ] `mvn test` of whole project
-- [ ] `mvn test` of submodule
-- [ ] `gradle test` (for the gradle plugin)
-- [ ] Integration tests with a running SW360
+> If you have added any changes that require additional tests, or changes in tests, you should implement them and describe them here.  
+> All test that passed before your contribution should pass after it as well. 
 
 ### Checklist
 Must:
 - [ ] Existing tests still pass with my changes
```

Comment:
I'd still argue that "[ ] Existing tests still pass with my changes" should also be removed because it's CI's responsibility to check for that.